ibis/backends/flink/tests/test_literals.py

```diff
@@ -0,0 +1,72 @@
+import datetime
+
+import pandas as pd
+import pytest
+from pytest import param
+
+import ibis
+import ibis.expr.datatypes as dt
+from ibis.backends.flink.compiler.core import translate
+
+
+@pytest.mark.parametrize(
+    "value",
+    [
+        param(5, id="int"),
+        param(1.5, id="float"),
+        param(True, id="true"),
+        param(False, id="false"),
+    ],
+)
+def test_simple_literals(snapshot, value):
```

Comment:
While snapshot tests can work here, for short cases like this where the expected value isn't a long or multi-line string, I slightly prefer manually writing out the full expected value in the parametrized test itself. For translating literals I'd expect this to be the case here.

Snapshot tests are nice when testing a larger query structure, where the output is larger and small changes in formatting (e.g. whitespace changes) can result in test failures. Snapshot tests make it easier in those cases to compare the failures with the new results and check that they're semantically the same.

Something like (untested):

```python
@pytest.mark.parametrize(
    "value, sol",
    [
        param(5, "5", id="int"),
        param(1.5, "1.5", id="float"),
        param(True, "TRUE", id="true"),
        param(False, "FALSE", id="false"),
    ],
)
def test_simple_literals(value, sol):
    expr = ibis.literal(value)
    result = translate(expr.op())
    assert result == sol
```

I'd do the same for all these literal tests. Definitely a style preference though, not a big deal if you disagree.

Reply:
No I think that's valid. Thanks for the feedback. I've made the changes.